pkg/rootless/rootless_linux.go

```diff
@@ -578,7 +578,6 @@ func TryJoinFromFilePaths(pausePidPath string, needNewNamespace bool, paths []st
 				return false, 0, err
 			}
 			defer func() {
-				errorhandling.CloseQuiet(r)
```

Comment:
I see why we're dropping the close on `w` above, but why `r` here?

Reply:
we already have a `defer errorhandling.CloseQuiet(r)` just after `r, w := os.NewFile(uintptr(fds[0]), "read file"), os.NewFile(uintptr(fds[1]), "write file")`

Reply:
we already have `defer errorhandling.CloseQuiet(r)` at line 570